File: 2nd_Course/Q1/POO/Unit_1/Unit_1_Practice/src/ex05/ex05.java
/*
 * Escribir un programa en Java que calcule el valor de la suma de todos los elementos de un array de enteros,
 *  e imprima el resultado por pantalla.
 * Nota: Podéis usar .length en un array para conocer su longitud
 * */
package ex05;

public class ex05 {
	public static void main(String[] args) {
		int[] num = {1, 2, 3, 4, 5};
		
		int sum = 0; 
		int i = 0; 
			
		while(i < num.length) {
			sum += num[i]; 
			i++; 
		}
		System.out.println("la suma de los elementos del array es: " + sum);
	}
}

File: 1st_Course/Q1/Programming_1/U2/Extra_Exercises/ex05.c
/*Ejercicio: Escribe un programa que imprima los números del 1 al 10 usando un while y luego con un for*/
#include <stdio.h>

int main() {
    int i = 1;

    printf("Usando while:\n");
    while (i <= 10) {
        printf("%d ", i);
        i++;
    }
    printf("\n");

    printf("Usando for:\n");
    for (i = 1; i <= 10; i++) {
        printf("%d ", i);
    }
    printf("\n");

    return 0;
}
